HMSegmentedControl/HMSegmentedControl.h

```diff
@@ -23,7 +23,7 @@ typedef NS_ENUM(NSInteger, HMSegmentedControlSelectionStyle) {
 typedef NS_ENUM(NSInteger, HMSegmentedControlSelectionIndicatorLocation) {
     HMSegmentedControlSelectionIndicatorLocationUp,
     HMSegmentedControlSelectionIndicatorLocationDown,
-	HMSegmentedControlSelectionIndicatorLocationNone // No selection indicator
```

Comment:
Not required, but it'd be nice if we could get rid of all these whitespace changes. If we ever want to rebase our fork back on the main repo, this could cause more conflicts, as well as making it harder to see what you intended to change.